src/Nethermind/Nethermind.Core/Transaction.cs

```diff
@@ -334,7 +334,7 @@ public class SystemTransaction : Transaction
     /// <remarks>Created because of cyclic dependencies between Core and Rlp modules</remarks>
     public interface ITransactionSizeCalculator
     {
-        int GetLength(Transaction tx);
+        int GetLength(Transaction tx, bool doNotCountBlobs = false);
```

Comment:
`doNotCountBlobs` here but `shouldCountBlobs` above - confusing. Also I would prefer positive over negative as it is easier to reason about.